docs/root/api-v1/network_filters/http_conn_man.rst

```diff
@@ -214,6 +214,12 @@ generate_request_id
   *true*. Generating a random UUID4 is expensive so in high throughput scenarios where this
   feature is not desired it can be disabled.
 
+proxy_100_continue
```

Comment:
You didn't implement v1 for this feature, right? Assuming not we don't want the docs here...